File: 2942-find-words-containing-character/2942-find-words-containing-character.ts
function findWordsContaining(words: string[], x: string): number[] {
    let indicies = []
    words.forEach((el, index)=>{
        for(let i = 0; i < el.length; i++){
            if(el[i] == x){
                indicies.push(index)
                break
            }
        }
    })
    return indicies
};

File: 2942-find-words-containing-character/2942-find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
       vector<int> answ;
       for(int i = 0; i < words.size(); i++){
        string s = words[i];
        for(char c : s){
            if(c == x){
                answ.push_back(i);
                break;
            }
        }
       } 
       return answ;
    }
};